content/en/docs/devops-user-guide/how-to-integrate/harbor.md

```diff
@@ -0,0 +1,128 @@
+---
+title: "How to integrate Harbor in Pipeline"
+keywords: 'kubernetes, docker, devops, jenkins, harbor'
+description: ''
+linkTitle: "Integrate Harbor in Pipeline"
+weight: 360
+---
+
+## Prerequisites
+
+- You need to [enable KubeSphere DevOps System](../../../../docs/pluggable-components/devops/).
+- You need to create a workspace, a DevOps project, and a **project-regular** user account, and this account needs to be invited into a DevOps project. See [create-workspace-and-project](../../../../docs/quick-start/create-workspace-and-project).
+- You need to have installed **Harbor** already. 
+
+## Install  Harbor
+
+It is highly recommended that you install harbor [by application store](). You can also install harbor manally by helm3.
```

Comment:
This link relies on **harbor-installation-docs**. When this [pr](https://github.com/kubesphere/website/pull/379) is merged, we can add the doc link into this doc.
@FeynmanZhou